### PR TITLE
[MIG] stock_reserve: Migration to 16.0

### PR DESCRIPTION
**Pull request by aliciagaarzo**

### CODE REVIEW

File: stock_reserve/__manifest__.py [hunk 1]
```diff
@@ -9,7 +9,7 @@
     "license": "AGPL-3",
     "complexity": "normal",
     "website": "https://github.com/OCA/stock-logistics-warehouse",
-    "depends": ["stock"],
+    "depends": ["stock", "sale_stock"],
```
Comment (rousseldenis) [inline stock_reserve/__manifest__.py]: @aliciagaarzo You cannot add a dependency just to fix tests :sweat_smile:

Comment (aliciagaarzo) [reply stock_reserve/__manifest__.py]: The tests fail me because I don't have the dependency and I added it but I remove it

**Comment by aliciagaarzo** [comment]
@rousseldenis Could you review the new change?

**Comment by rousseldenis** [comment]
Seems great. Could you rearrange your commits (squash all together except the pre-commit one that should go with the other one) ?

**Comment by aliciagaarzo** [comment]
> Seems great. Could you rearrange your commits (squash all together except the pre-commit one that should go with the other one) ?
Sure, I change it!

**Review by rousseldenis — APPROVED** [review]
Code review

**Comment by hugo-cordoba** [comment]
LGTM! (Code review)

**Review by hugo-cordoba — APPROVED** [review]

**Pull request merged**